script/ensure-node-source-fixture-files.js

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const path = require('path');
+const https = require('https');
```

Comment:
```suggestion
const fs = require("fs");
const path = require("path");
const https = require("https");
```
Codacy suggestion: Single quote -> Double quote